### ISSUE TITLE
Webdav Sync with `http://` silently fails with success messages in the UI

### ISSUE THREAD

**Issue by rhnvrm** [orgzly/orgzly-android#682]
- [x] I have searched for existing issues that may be the same as or related to mine.
Hey

I have a self hosted nextcloud instance.

I added a repository with `http://` instead of `https://` and in the UI the sync shows that sync was completed without any error messages. Although, nothing actually changes in the server.

After switching to `https://` I had to delete the notebooks and resync again, and everything went smoothly.

**Comment by nevenz** [comment]
And `http` URL points to the same Nextcloud server? Nothing is done differently, results should be the same. Can you reproduce this every time (assuming you tried it again after switching to https)?

> After switching to https:// I had to delete the notebooks and resync again, and everything went smoothly.

This is currently necessary as it's a different repo. It could be handled better. Alternatively, you could have set (changed) a link for every notebook.

**Comment by rhnvrm** [comment]
Yes, the `http` URL points to the same `https://` URL and I could reproduce it as well back then as I was testing the integration. Local changes were fine but they were not getting pushed to the server.

**Comment by nevenz** [comment]
I'd have to setup a server with https access to debug this if you don't have one I can use. Or perhaps adb logs have something useful if you can see those.

**Comment by rhnvrm** [comment]
I'll try to share the adb logs in some time.
